lib/bin/cli.js

```diff
@@ -16,23 +16,33 @@
 const { run } = require('../task/task');
 const { createUser, promoteUser, setUserPassword } = require('../task/account');
 
+const crypto = require('node:crypto');
+
 // gets a password interactively if not supplied in cli args.
 const prompt = require('prompt');
-const withPassword = (f) => {
-  prompt.start();
-  prompt.get([{ name: 'password', hidden: true, replace: '*' }], (_, { password }) => f(password));
-};
 
 // command line nonsense (i'm not a huge fan of this library).
 const cli = require('cli');
 const cliArgs = {
-  email: [ 'u', 'For user create and set password commands, supplies the email.', 'email' ]
+  email: [ 'u', 'For user create and set password commands, supplies the email.', 'email' ],
+  'generate-password': [ 'g', 'For user create and set password commands, generates a random password and prints it to stdout.', 'boolean' ],
 };
 const cliCommands = [ 'user-create', 'user-promote', 'user-set-password' ];
 cli.parse(cliArgs, cliCommands);
 
 // map commands to tasks.
 cli.main((args, options) => {
+  const withPassword = (f) => {
+    if (options['generate-password']) {
+      const password = crypto.randomBytes(32).toString('base64');
+      console.log(password);
```

Comment:
Output with `-g` flag is like this:

```
9BonCXVsfn/tQfmp4c3+xrsTlGQzm+8aQ9UCnEeqkKs=
{
  email: 'sadiq2@gmail.com',
  id: 72,
  type: 'user',
  displayName: 'sadiq2@gmail.com',
  createdAt: 2023-07-11T16:09:59.160Z,
  updatedAt: null,
  deletedAt: null
}
```

Can we beautify the output by either moving password inside the output object or by adding a label in front of the password

Reply:
I have moved the password output to stderr, and added `Password: ` prefix to make clear what is being printed.